lib/actions/api.js

```diff
@@ -328,6 +358,76 @@ export function findStop (params) {
   )
 }
 
+export function fetchStopInfo (stop) {
+  return async function (dispatch, getState) {
+    await dispatch(findStop({ stopId: stop.stopId }))
+    const state = getState()
+
+    const fetchedStop = state.otp.transitIndex.stops[stop.stopId]
+    const {lat, lon} = fetchedStop
+    const radius = 250
```

Comment:
This should either be a file-wide constant or a config value

Reply:
It's only used in the two lines below. I'm against file-wide constants for variables such as this.

I'll add as config value though.